vue/components/ui/molecules/lineup/lineup.stories.js

```diff
@@ -50,7 +50,7 @@ storiesOf("Molecules", module)
                 <template v-slot:shirt>
                     <p>Custom entry</p>
                 </template>
-                <template v-slot:city-label>
+                <template v-slot:city>

```

Comment:
Why?

Reply:
It wasn't appearing, because it doesn't match the naming. Don't know if that was the original purpose though

Reply:
I've checked and it's the `lineup` who's wrong: `v-bind:name="`${value.value}-key`"` should be `v-bind:name="`${value.value}-label`"`